### ISSUE TITLE
How to change the default terminal

### ISSUE THREAD

**Issue by Destroyer-LDB** [microsoft/terminal#7159]
# **Set the command prompt as the default terminal**

I am new to use Windows Terminal, and I find that when I open a new terminal, it is the Windows Powershell. But actually, I prefer to use the command prompt. However, I don't know how to change the default terminal. I read the document, but I found nothing about it in the document. Can someone tell me how to set the command prompt as the default terminal instead of Windows Powershell?

**Comment by zadjii-msft** [comment]
Wait are you talking about changing the default shell on the Win+X menu? 

![image](https://user-images.githubusercontent.com/18356694/89192545-3b845780-d56a-11ea-9c8c-8f4f8594287e.png)

You might find [this](https://www.google.com/search?q=win+x+command+prompt) helpful.

**Comment by zadjii-msft** [comment]
Or are you talking about changing the `defaultProfile` of the Windows Terminal? From [the docs](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/global-settings):

> To customize the settings of your Windows Terminal, select Settings in the dropdown menu. This will open the `settings.json` file in your default text editor. (The default text editor is defined in your Windows settings.)

> ### Default profile
> Set the default profile that opens by typing <kbd>ctrl+shift+t</kbd>, typing the key binding assigned to `newTab`, running `wt new-tab` without specifying a profile, or clicking the '+' icon.
> 
> Property name: `defaultProfile`

**Comment by JanAnders** [comment]
Seeing that you are still in school, you may have never worked with config files before and this may seem a bit strange at first, but you'll get the hang of it eventually. Follow these instructions.

Open the terminal settings file by hitting Ctrl + , or select "Settings" in the dropdown menu. A text file will open.

See the line where it says "defaultProfile"? It is the guid (the random string of letters) inside the curly brackets that you want to change to the guid of the profile (e.g. power shell, cmd, azure shell, ...) that is opened when opening a new terminal tab.

In the settings document, scroll down to the "profiles" section. If you have changed no other settings, this should be only a few lines below. Search for the profile of the Command Prompt (where "commandline": "cmd.exe") and copy the value of the "guid" parameter. Overwrite the guid parameter of the defaultProfile above with this value.

Save your changes by hitting Ctrl + s. Now any new tab you open should be the command prompt.

**Comment by Destroyer-LDB** [comment]
@zadjii-msft Well, I meant **change the defaultProfile of the Windows Terminal**.
I thought this setting should be called ''defaultTerminal'' or something like this.

**Comment by Destroyer-LDB** [comment]
@JanAnders Thank you! Your way really works. Well, I have worked with config files before ( also JSON ). It is the settings.json of Visual Studio Code. But VSC has a GUI page for the ''Settings'' command. When I clicked the ''Settings'' in Windows Terminal for the first time, I felt surprised because no settings window showed in my screen, but a VSC window opening the settings.json. So I think I will open another issue  asking for a GUI to edit settings. Thank you for telling me how to config the defaultProfile.
